lib/system/strs_v2.nim

```diff
@@ -36,7 +36,7 @@ template frees(s) =
 
 proc resize(old: int): int {.inline.} =
   if old <= 0: result = 4
-  elif old < 65536: result = old * 2
+  elif old <= high(int16): result = old * 2
```

Comment:
That's definitely high(uint16) == 65535

Reply:
Why? I don't see what's wrong in the former implementation.

Reply:
`int` now is `int16` when target 16bit system which is `32767`